www/index.rst

```diff
@@ -28,24 +29,48 @@ few restrictions.
 Getting Started
 ---------------
 
-- `Getting NumPy <http://www.scipy.org/scipylib/download.html>`__
-- `Installing the SciPy Stack <http://www.scipy.org/install.html>`__
-- `NumPy and SciPy documentation page <http://docs.scipy.org/doc/>`__
-- `NumPy Tutorial <https://docs.scipy.org/doc/numpy/user/quickstart.html>`__
-- `NumPy for MATLAB© Users <https://docs.scipy.org/doc/numpy/user/numpy-for-matlab-users.html>`__
-- `NumPy functions by category <https://docs.scipy.org/doc/numpy/reference/routines.html>`__
-- `NumPy Mailing List <http://www.scipy.org/scipylib/mailing-lists.html>`__
+- :ref:`obtaining`
+- `NumPy Tutorial <devdocs/user/quickstart.html>`__
+- `NumPy for MATLAB© Users <devdocs/user/numpy-for-matlab-users.html>`__
+- `NumPy functions by category <devdocs/reference/routines.html>`__
+- `NumPy Mailing List <https://mail.python.org/mailman/listinfo/numpy-discussion>`__
 
 For more information on the SciPy Stack (for which NumPy provides the
-fundamental array data structure), see `scipy.org <http://www.scipy.org/>`__.
+fundamental array data structure), see `scipy.org <https://www.scipy.org/>`__.
+
+
+Documentation
+-------------
+
+`Latest (development) version <devdocs>`__
+
+* `Version 1.15.4 <https://docs.scipy.org/doc/numpy-1.15.4>`__,
+  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.15.4/numpy-html-1.15.4.zip>`__
+  `[PDF] <https://docs.scipy.org/doc/numpy-1.15.4/numpy-ref-1.15.4.pdf>`__
+* `Version 1.15.1 <https://docs.scipy.org/doc/numpy-1.15.1>`__,
```

Comment:
I suggest only to put the `.0` and the `.most-recent` links, the rest is just noise.

Reply:
oh yes, and can you add 1.16.x?

Reply:
I will remove the noise. One of the goals of this is to locally host the 1.16.x docs on numpy.org, so I would like to add them as a second PR once we have a way to upload them there

Reply:
The `.0` versions or the latest in the series (1.14.5, 1.15.4, ...)?

Reply:
Now reduced to 1.16.2, 1.15.4, 1.14.5, 1.13.0

Reply:
thanks, looks better